src/app/portlets/dot-edit-page/content/components/dot-whats-changed/dot-whats-changed.component.ts

```diff
@@ -1,18 +1,18 @@
-import { Component, OnInit, Input } from '@angular/core';
+import { Component, OnInit, Input, OnChanges } from '@angular/core';
 
 @Component({
     selector: 'dot-whats-changed',
     templateUrl: './dot-whats-changed.component.html',
     styleUrls: ['./dot-whats-changed.component.scss']
 })
-export class DotWhatsChangedComponent implements OnInit {
+export class DotWhatsChangedComponent implements OnChanges {
     @Input() languageId: string;
     @Input() pageId: string;
     url: string;
 
     constructor() {}
 
-    ngOnInit() {
+    ngOnChanges(): void {
```

Comment:
should this be executed when anything changes? or only when something specific changes?

Reply:
when anything change